.kokoro/build.sh

```diff
@@ -112,6 +112,28 @@ clirr)
     mvn -B -Denforcer.skip=true clirr:check
     RETURN_CODE=$?
     ;;
+retry_quota)
+    mvn -B ${INTEGRATION_TEST_ARGS} \
+      -Penable-integration-tests \
```

Comment:
Enabling this profile will run all the other IT tests also. Do we need that?

Reply:
We do not; I changed the command to test with explicit -Dtest args, let me know if that still needs work